js/specificFilters.js

```diff
@@ -628,8 +628,8 @@ class StaircaseFilter extends BooleanFilter {
 class FormFilter extends OptionsFilter {
     constructor(api) {
         var name = 'form';
-        var label = 'Form of the sound';
-        var options = ['click','pipe trains','prolonged','SAM (sinusoidal amplitude modulation)'];
+        var label = 'Test tone presentation';
+        var options = ['click','pip trains','prolonged'];
```

Comment:
Please also delete "prolonged" - when reviewing the DB we could not substantiate this category and there aren't any datasets matching this one.